### PR TITLE
Add logs filtered files and editorConfig file

### PR DESCRIPTION
**Pull request by nebur395**
- Added logs filtered files. Now, there are three ways to see the loggers:
   - **Console**: All loggers are displayed.
   - **logs/app_log.log file**: App loggers are displayed.
   - **logs/springboot_log.log file**: Spring loggers are displayed.
- Added EditorConfig file and some interesting plugins of it in description.md file.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# http://EditorConfig.org
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
```
Comment (RecuencoJones) [inline .editorconfig]: Repeated pattern [*] either create more specific targets or place them together

File: description.md [hunk 2]
```diff
@@ -139,6 +139,12 @@ In our case, our project has the next structure:
 
 As we can see our project follows Mavens directory layout. There are some directories in the layout that aren't present in our project structure but this is because our project source doesn't use anything from those directories since they are generated at building time.
 
+# Plugins don't supported by EditorConfig file
```
Comment (RecuencoJones) [inline description.md]: What? Should be plugins/IDEs supporting Editorconfig?

**Comment by nebur395** [comment]
Done

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (SAReyes) [inline .editorconfig]: Please create a more specific target for these rules.

Comment (nebur395) [inline .editorconfig]: Don't we want it for all files in the project?

Comment (SAReyes) [inline .editorconfig]: I wrote this comment on a different line than the one I was thinking of, my bad. I was referring to line 13, the indent size should be specific to java, jsp, xml, etc.

Comment (nebur395) [inline .editorconfig]: Done :octocat:

File: description.md [hunk 4]
```diff
@@ -139,6 +139,12 @@ In our case, our project has the next structure:
 
 As we can see our project follows Mavens directory layout. There are some directories in the layout that aren't present in our project structure but this is because our project source doesn't use anything from those directories since they are generated at building time.
 
+# IDEs not supported by EditorConfig file
```
Comment (fjlopez) [inline description.md]: Add a section explaining what is EditorConfig and which IDE supports (i.e. a summary of http://editorconfig.org/ _What is EditorConfig?_ and _No Plugin Necessary_)

Comment (nebur395) [inline description.md]: Done

**Pull request merged**